### PR TITLE
A few ways to hack stuff on GitHub

### PR DESCRIPTION
**Pull request by bburky**

### CODE REVIEW

File: generator/main.ts [hunk 1]
```diff
@@ -152,7 +152,9 @@ async function main() {
 
     const routeDir = path.join("public", slug)
 
-    fs.mkdirSync(routeDir)
+    if (!fs.existsSync(routeDir)){
+      fs.mkdirSync(routeDir)
+    }
```
Comment (bburky) [inline generator/main.ts]: This supports adding images to a post in a `/static/<slug>` dir.

File: generator/md.ts [hunk 2]
```diff
@@ -69,8 +69,7 @@ function extractHeadersPlugin(md: markdownit): void {
     if (titleToken.type === "inline") {
       const title =
         titleToken.children
-          ?.filter((t) => t.type === "text")
```
Comment (bburky) [inline generator/md.ts:72]: My headings had `<code>` tags. This seems to fix it. The text of any tags will be concatenated together. Don't restrict to "text" only, I think that's just text not in tags.

File: generator/main.ts (hunk 1 above)
Comment (Noxsios) [reply generator/main.ts]: I'll update `content/tutorial.md` w/ that info.

File: generator/md.ts (hunk 2 above)
Comment (Noxsios) [reply generator/md.ts:72]: 99% of that function is GPT4 + copilot xD

**Review by Noxsios — APPROVED** [review]

**Pull request merged**